### PR TITLE
add synthetics configuration docs

### LINKED ISSUE #846: [Synthetics] Docs on synthetics config file
+ We recently added support in the synthetics agent which users can control to add all the playwright specific browser and context options. 

We need to document the same.

### PR DESCRIPTION
**Pull request by vigneshshanmugam**
+ fixes #846

### CODE REVIEW

File: docs/en/observability/synthetics-configuration.asciidoc [hunk 1]
```diff
@@ -0,0 +1,64 @@
+[[synthetics-configuration]]
+= Synthetic tests configuration
+
+++++
+<titleabbrev>Synthetic tests configuration</titleabbrev>
+++++
+
+beta[] Synthetic tests support the configuration of dynamic parameters that can be
+used in the suites. In addition to that, the Synthetics agent which is built on top
+of Playwright supports configuring browser and context options that are available
```
Comment (bmorelli25) [inline docs/en/observability/synthetics-configuration.asciidoc:10]:
```suggestion
used in the suites. In addition, the Synthetics agent, which is built on top
of Playwright, supports configuring browser and context options that are available
```

File: docs/en/observability/synthetics-configuration.asciidoc [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+in Playwright specific methods, for example, `ignoreHTTPSErrors`, `viewport`.
```
Comment (bmorelli25) [inline docs/en/observability/synthetics-configuration.asciidoc:11]:
```suggestion
in Playwright specific methods, for example, `ignoreHTTPSErrors` and `viewport`. 
```

File: docs/en/observability/synthetics-configuration.asciidoc [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+used in the suites. In addition to that, the Synthetics agent which is built on top
+of Playwright supports configuring browser and context options that are available
+in Playwright specific methods, for example, `ignoreHTTPSErrors`, `viewport`. 
+
+[discrete]
+[[synthetics-config-file]]
+== Global Synthetics Configuration
+
+Create `synthetics.config.js` or `synthetics.config.ts` in the root of
+synthetics project and specify the options
```
Comment (bmorelli25) [inline docs/en/observability/synthetics-configuration.asciidoc:18]:
```suggestion
Create a `synthetics.config.js` or `synthetics.config.ts` file in the root of the
synthetics project and specify the options:
```

File: docs/en/observability/synthetics-configuration.asciidoc [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  },
+  playwrightOptions: {
+    ignoreHTTPSErrors: true, // ignores all HTTPS errors during navigation
+  }
+}
+
+export default config;
+----
+
+The configuration file can return either a function or an object which will be
```
Comment (bmorelli25) [inline docs/en/observability/synthetics-configuration.asciidoc:36]: This sentence looks unfinished

File: docs/en/observability/synthetics-configuration.asciidoc [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+To know more about configuring the tests based on environments, look at the
+<<synthetics-dynamic-configs, dynamic configuration>> documentation.
```
Comment (bmorelli25) [inline docs/en/observability/synthetics-configuration.asciidoc:39]:
```suggestion
To learn more about configuring synthetic tests based on environments, see
<<synthetics-dynamic-configs, dynamic configuration>>.
```

File: docs/en/observability/synthetics-configuration.asciidoc [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+[discrete]
+[[synthetics-config-device-emulation]]
+=== Device emulation
+
+Users can emulate a mobile device using the configuration file. Check
+the example configuration below that runs tests in "Pixel 5" emulation mode.
```
Comment (bmorelli25) [inline docs/en/observability/synthetics-configuration.asciidoc:46]:
```suggestion
Users can emulate a mobile device using the configuration file.
The example configuration below runs tests in "Pixel 5" emulation mode.
```

File: docs/en/observability/synthetics-configuration.asciidoc (hunk 4 above)
Comment (vigneshshanmugam) [reply docs/en/observability/synthetics-configuration.asciidoc]: Fixed.

**Review by bmorelli25 — APPROVED** [review]

**Pull request merged**